.github/workflows/continuous-integration.yaml

```diff
@@ -9,6 +9,33 @@ on:
       - 'alpha-release/**'
 
 jobs:
+  autopep8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: autopep8
+        id: autopep8
+        uses: peter-evans/autopep8@v1
+        with:
+          # ignore E402 rule to avoid changes that cause circular imports
+          args: --exit-code --recursive --in-place --aggressive --aggressive --ignore E402 .
```

Comment:
You can see all the rules and arguments that can be used [here](https://github.com/hhatto/autopep8)